assets/blocks/restricted-content/settings.js

```diff
@@ -0,0 +1,140 @@
+import {
+	BlockControls,
+	BlockSettingsMenuControls,
+} from '@wordpress/block-editor';
+import {
+	Button,
+	Dropdown,
+	NavigableMenu,
+	Toolbar,
+	MenuItem,
+} from '@wordpress/components';
+import { RestrictOptions, RestrictOptionLabels } from './edit';
+import { __ } from '@wordpress/i18n';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Check if the current block is the only one that is selected.
+ *
+ * @param {string} clientId The block client id.
+ */
+const useIsSingleRestrictSelected = function ( clientId ) {
```

Comment:
Could we change these functions to arrow functions, just to keep the standard? :)

Reply:
254ed678535a1e3ac3beb108a9d70f2e42a77bbe

Reply:
I think we missed the `useOnRestrictionRemoval` too. :)

Reply:
Ah! Missed that 216adca3b33831d36867e2ed4175917b321e8fb4